tests/io_utils.go

```diff
@@ -117,11 +120,46 @@ func FixErrorDevice(nodeName string) {
 
 }
 
+func executeDeviceMapperOnNode(nodeName string, cmd []string) {
```

Comment:
Just noted that `nodeName` is not used in this function. Perhaps an affinity rule is missing here?

Reply:
oh, thanks!